tests/unit/dependencies.js

```diff
@@ -0,0 +1,167 @@
+(function () {
```

Comment:
What I mean is that if every injectable in this package would just be under one module `angular-token-auth` then testing the availability wouldn't be necessary at all. It doesn't allow to specify which modules you do or don't want when using this package, but on the other hand maybe all injectables that require each other should be just under one module?